docs/setup/aws.md

```diff
@@ -1,107 +1,148 @@
+### Clone Grapl Git repository
 
 ```bash
-$ cat .env
-TAG="v0.0.1-example"
-$ make zip
+git clone https://github.com/grapl-security/grapl.git
+cd grapl/
 ```
 
-Similar can be done for the environment variables corresponding to CDK
-deployment parameters documented in the following section.
+The remaining steps assume your working directory is the Grapl repository.
 
-Your build outputs should appear in the `src/js/grapl-cdk/zips/` directory.
+### Build deployment artifacts
 
-## Configure
+Deployment artifacts are build via `make zip`.
 
-There are a few CDK deployment parameters you'll need to configure before you can deploy.
-Each of these can be found in `bin/deployment_parameters.ts`:
+If environmental variable `TAG` is set, it will be a custom name for the build.  If it is unset, it will default to "latest".
 
-1. `GRAPL_DEPLOYMENT_NAME` (required)
-
-    Name for the deployment to AWS. We recommend prefixing the
-    deployment name with "Grapl-" to help identify Grapl resources in
-    your AWS account, however this isn't necessary.
+```bash
+# build with 'latest' TAG:
+make zip
 
-    Note: This name must be globally (AWS) unique, as names for AWS S3
-    buckets will be dervied from this.
+# build with a one-off custom TAG:
+$ TAG=my_grapl_test make zip
 
-2. `GRAPL_VERSION`
+# or set a fixed custom TAG in .env:
+$ cat .env
+TAG="my_grapl_test"
+$ make zip
+```
 
-    The version of Grapl to deploy. This string will be used to look
-    for the appropriate filenames in the `zips/` directory.
+After `make zip` finishes, you can inspect `src/js/grapl-cdk/zips/` to see the build outputs:
 
-    Defaults to `latest`.
+```bash
+ls ./src/js/grapl-cdk/zips/
+```
 
-3. `GRAPL_CDK_WATCHFUL_EMAIL` (optional)
+They should be named according to the value of `TAG`.
 
-    Setting this enables [Watchful](https://github.com/eladb/cdk-watchful) for
-    monitoring Grapl with email alerts.
+### Configure CDK deployment parameters
 
-4. `GRAPL_CDK_OPERATIONAL_ALARMS_EMAIL` (optional)
+Grapl's CDK deployment parameters are set as environmental variables.
 
-    Setting this enables alarms meant for the operator of the Grapl stack.
+For a direct code reference, c.f. `src/js/grapl-cdk/bin/deployment_parameters.ts`.
 
-5. `GRAPL_CDK_SECURITY_ALARMS_EMAIL` (optional)
+#### Recommended approach
 
-    Setting this enables alarms meant for the consumer of the Grapl
-    stack, for example, "a new risk node has been found".
+If you are deploying to production, we recommend saving a `source`-able
+collection of these environment variables:
 
-When deploying to production we recommend creating a `source`-able 
-collection of these environment variables, and saving that to some
-version control; something along the lines of the following:
 ```bash
-export GRAPL_ROOT="~/src/grapl"
+# example values
 export GRAPL_DEPLOYMENT_NAME="some-grapl-deployment-name"
 export GRAPL_VERSION="latest"
+export GRAPL_ROOT="~/src/grapl"
```

Comment:
I know these are example variables, and this was a pre-existing value, but it would be good to make this more obviously fake, like `/path/to/wherever/you/cloned/the/grapl/repository`.

`~/src/grapl` is not obviously fake, and may actually work for some people, but not everyone has their directories laid out in the same way (like me, for one 😅). If someone accidentally used this value, it could lead to a more frustrating debugging experience.